dev/fixGrammarTypes.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs-extra');
+const path = require('path');
+const readline = require('readline');
+const EOL = require('os').EOL;
```

Comment:
These `require` statements are making lint fail.  I was going to say to change them to `import` statements -- but then I think you need to run it through babel for node to be able to process it.

So, I think your choices are to turn off this eslint rule for this file or to convert the file to `.ts` and use `ts-node` in the npm task instead of `node`.  Since this is a TypeScript project already, I kind of lean toward the latter.